.travis.yml

```diff
@@ -7,12 +7,16 @@ go:
   - 1.6
 
 before_install:
+ - go get -u github.com/coreos/git-validation
+ - git-validation -run subsystem-in-subject,short-subject,dangling-whitespace
+
+install:
```

Comment:
Note that I moved the fail-fast tests to `before_install` and the preparation to `install`